CHANGELOG.md

```diff
@@ -8,6 +8,22 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 ## [next-version]
 
+## Added
+
+- Internal: Added unit tests for Demo and App components
+
+### Changed
+
+- Public: Improve description of `showCountrySelection` option for Document step to be more explicit about when/how it works and include example configurations.
+- Internal: Store third-party licence comments for each bundle in separate files.
+- UI: Accessibility - Update passport quality guide copy to be more descriptive for visually impaired users using screen readers
+
+### Fixed
+
+- UI: Accessibility - Loading screen is now announced on iOS
+- Internal: Release script didn't update `BASE_32_VERSION` correctly and didn't finish at publishing tag step
```

Comment:
These logs have been moved out of the version they belong to 🤔

Reply:
I think that would be because the version they're in has not been released yet (in progress still) so it wouldn't have been merged in yet.

Reply:
I've updated the latest changelog from #1220, seems that it's the only way to sort all of these things out.